packages/file_selector/file_selector_web/lib/src/dom_helper.dart

```diff
@@ -14,50 +14,51 @@ class DomHelper {
 
   /// Default constructor, initializes the container DOM element.
   DomHelper() {
-    final body = querySelector('body');
+    final body = querySelector('body')!;
     body.children.add(_container);
   }
 
   /// Sets the <input /> attributes and waits for a file to be selected.
   Future<List<XFile>> getFiles({
     String accept = '',
     bool multiple = false,
-    @visibleForTesting FileUploadInputElement input,
+    @visibleForTesting FileUploadInputElement? input,
   }) {
-    final Completer<List<XFile>> _completer = Completer();
-    input = input ?? FileUploadInputElement();
+    final Completer<List<XFile>> completer = Completer();
+    final FileUploadInputElement inputElement =
+        input ?? FileUploadInputElement();
 
     _container.children.add(
-      input
+      inputElement
         ..accept = accept
         ..multiple = multiple,
     );
 
-    input.onChange.first.then((_) {
-      final List<XFile> files = input.files.map(_convertFileToXFile).toList();
+    inputElement.onChange.first.then((_) {
+      final List<XFile> files = input!.files!.map(_convertFileToXFile).toList();
       input.remove();
-      _completer.complete(files);
+      completer.complete(files);
     });
 
-    input.onError.first.then((event) {
-      final ErrorEvent error = event;
+    inputElement.onError.first.then((event) {
+      final ErrorEvent error = event as ErrorEvent;
       final platformException = PlatformException(
         code: error.type,
         message: error.message,
       );
-      input.remove();
-      _completer.completeError(platformException);
+      input!.remove();
+      completer.completeError(platformException);
     });
 
-    input.click();
+    inputElement.click();
 
-    return _completer.future;
+    return completer.future;
   }
 
   XFile _convertFileToXFile(File file) => XFile(
         Url.createObjectUrl(file),
         name: file.name,
         length: file.size,
-        lastModified: DateTime.fromMillisecondsSinceEpoch(file.lastModified),
+        lastModified: DateTime.fromMillisecondsSinceEpoch(file.lastModified!),
```

Comment:
Why is this safe?

Reply:
I think this is `?` in the definition because of older browsers that might not expose this feature [see docs](https://developer.mozilla.org/en-US/docs/Web/API/File/lastModified).

I've defaulted this to `DateTime.now().millisecondsSinceEpoch`, which is the default for files that don't have this field according to MDN.